onnxruntime/core/framework/stream_execution_context.h

```diff
@@ -37,7 +37,7 @@ class StreamExecutionContext {
    public:
     CountDownBarrier() : v_{0} {};
 
-    void Set(int32_t v) {
+    void Set(int_fast32_t v) {
```

Comment:
Not clear how this is related to enabling the warning

Reply:
I redid this change. the original change was to make the Get()/Set() types consistent with the atomic's underlying type (int_fast32_t), which was possibly wider than int32_t.